_includes/markdown/Markup.md

```diff
@@ -279,9 +180,12 @@ These are not easily maintainable and can be easily lost or cause unforeseen con
 <h2 id="accessibility">Accessibility {% include Util/top %}</h2>
 It's important that our clients and their customers are able to use the products that we create for them. Accessibility means creating a web that is accessible to all people: those with disabilities and those without. We must think about people with visual, auditory, physical, speech, cognitive and neurological disabilities and ensure that we deliver the best experience we possibly can to everyone. Accessibility best practices also make content more easily digestible by search engines. Increasingly, basic accessibility can even be a legal requirement. In all cases, an accessible web benefits everyone.
 
-At minimum, every 10up project should make use of ARIA Landmark roles, semantic headings, and alt text on images. Compliance with Section 508, or other international accessibility laws and guidelines, may be required depending on the project.
+### Accessibility Standards
+At a minimum, all 10up projects should pass [WCAG 2.0 Level A Standards](https://www.w3.org/WAI/intro/wcag). A baseline compliance goal of Level A is due to [WCAG guideline 1.4.3](https://www.w3.org/WAI/WCAG20/quickref/#visual-audio-contrast-contrast) which requires a minimum contrast ratio on text and images, as 10up does not always control the design of a project. If a third party design presents an inherit accessibility error, that concern should be brought to the client and project team. 
+
+For design projects and projects with a global marketplace, Level AA should be the baseline goal. The accessibility level is elevated for global markets to properly comply with [EU Functional Accessibility Requirements](http://mandate376.standards.eu/standard/technical-requirements/#9), which aligns closely with WCAG 2.0 Level AA. Having direct access to the designer also allows for greater accessibility standards to be achieved.
```

Comment:
What do you mean "global marketplace"?

Reply:
@tlovett1 Having entities outside of the US, like a company based in Europe/Asia where extra compliance is required.

Reply:
Updated that sentence with more clarity